test/Microsoft.Azure.Devices.Edge.Test/helpers/ManualProvisioningFixture.cs

```diff
@@ -7,6 +7,92 @@ namespace Microsoft.Azure.Devices.Edge.Test.Helpers
     using Microsoft.Azure.Devices.Edge.Test.Common;
     using NUnit.Framework;
 
+    public class X509ManualProvisioningFixture : ManualProvisioningFixture
+    {
+        public X509Thumbprint thumbprint { get; set; }
+        public EdgeDevice device { get; set; }
+
+        public X509ManualProvisioningFixture()
```

Comment:
Is this constructor necessary?

Reply:
You're right, I was under the impression that I needed to explicitly call the base constructor. TIL that it's automatically called when it's parameterless